content/sensu-go/6.2/operations/control-access/ldap-auth.md

```diff
@@ -144,6 +144,56 @@ spec:
 
 {{< /language-toggle >}}
 
+**Example LDAP configuration: Use `memberOf` attribute instead of `group_search`**
+
+LDAP automatically returns a `memberOf` attribute in users' accounts.
```

Comment:
So this is not true; by default most LDAP servers will not return a `memberOf` attribute unless they are explicitly configured to do so; Only Active Directory is configured by default to do so AFAIK

Reply:
@palourde got it. Does that mean that for LDAP, Sensu users need to both remove the group_search attribute and add a memberOf attribute? If so, what does the memberOf attribute look like?

Reply:
So the `memberOf` attribute is something returned by LDAP when you perform a query; this is not a configuration option in Sensu.